style.css

```diff
@@ -1 +1,116 @@
-/*  Add your styling here */
+body {
+  width: 590px;
+  margin-top: 3em;
+  margin-left: 7em;
+  margin-right: 3em;
+}
+/* header */
+img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 0px 0px;
+  grid-template-areas:
+    "a n"
+    "a s";
+
+  background-color: blanchedalmond;
+  width: 100%;
+  max-width: 590px;
+  margin: 0 auto;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.nav_menu {
+  list-style: none;
+}
+
+.nav_link {
+  text-decoration: none;
+}
+
+/* header-cake-shop */
+
+.cake_shop_avatar {
+  width: 7em;
+  height: 9em;
+  margin-top: 1em;
+  margin-left: 2em;
+
+  background-image: url(img/cupcake.webp);
+  background-size: cover;
+  background-position: center;
+}
+
+.navbar {
+  /* grid-row-start: 1;
+  grid-row-end: 2;
+  width: 100%; */
+  grid-area: n;
+  margin-bottom: 4em;
+  text-align: right;
+}
+
+.cake_shop_slogan {
+  /* grid-row-start: 2;
+  grid-row-end: 3; */
+  grid-area: s;
+  margin-top: -4em;
+  margin-left: 3em;
+  margin-right: -3em;
+  margin-bottom: 0em;
+  padding-left: 10em;
+  padding-right: 0em;
+  font-size: 8px;
+}
+
+/* hero */
+.hero {
```

Comment:
Why is this called hero?

Reply:
I used it because I thought it was the important section. Was it wrong?

Reply:
This term is new to me in this context, thus my question.

I googled and found the answer: https://www.awebco.com/blog/hero-section/

all good